### PR TITLE
fix: PHP add workflow retries

### LINKED ISSUE #1452: [Features] Workflow Retry Policy is missing in PHP
# [Workflow Retry Policy](https://docs.temporal.io/application-development/features?lang=php#workflow-retry-policy) is missing is PHP
 
---
You can provide a code snippet in the comments or a link to the line numbers of the samples-repo. Once added, I'll update the app-dev page with the new code snippet.

### PR DESCRIPTION
**Pull request by seregazhuk**
Resolves #1452

### CODE REVIEW

File: docs/application-development/features.md [hunk 1]
```diff
@@ -2518,7 +2518,20 @@ Content is currently unavailable.
 </TabItem>
 <TabItem value="php">
 
-Content is currently unavailable.
+Retry policy can be configured with an instance of `RetryOptions` object. To enable retries for a Workflow, 
+you need to provide a retry policy object via `ChildWorkflowOptions` for child Workflows or
+via `WorkflowOptions` for top-level Workflows:
```
Comment (flossypurse) [inline docs/application-development/features.md:2523]:
```suggestion
A Retry Policy can be configured with an instance of the `RetryOptions` object.
To enable retries for a Workflow,  you need to provide a Retry Policy object via `ChildWorkflowOptions` for child Workflows or via `WorkflowOptions` for top-level Workflows.
```

**Review by flossypurse — APPROVED** [review]

**Review by flossypurse — CHANGES_REQUESTED** [review]
Just realized that this is being added to the app dev guide directly.

This content needs to be added as `/php/set-a-workflow-retry-policy` or something similar.

**Comment by seregazhuk** [comment]
@flossypurse oops, fixed this one, and all my previous PR-s ✅

**Review by flossypurse — APPROVED** [review]

**Comment by flossypurse** [comment]
awesome @seregazhuk ! lgtm!

**Pull request merged**